### PR TITLE
feat(dashboard): back-pressure rate on stream graph in Grafana

### PR DESCRIPTION
**Pull request by BugenZhao**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

So that we can quickly find the bottleneck of a stream job. Inter-table flow chart is coming soon.

Preview:

image

Compared with #12397:

Pros:
- Embedded in Grafana. Intuitive and easy to access.
- Benefit from the Grafana(Prometheus)'s capability of handling timeseries. For example, inspect the back-pressure and throughput in a time-travelling way and show different indicators (mean vs p90 vs last) with familiar operations.

Cons:
- The flowchart is provided by a plugin. Provisioning requires extra work.
- The dev dashboard has much more details that allows us to inspect the original protobuf definition of job, internal table, or plan node body. So it may not be easily replaced.

## Checklist

- [ ] I have written necessary rustdoc comments
- [ ] I have added necessary unit tests and integration tests
- [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).
- [ ] My PR contains breaking changes. (If it deprecates some features, please create a tracking issue to remove them in the future).
- [ ] All checks passed in `./risedev check` (or alias, `./risedev c`)
- [ ] My PR changes performance-critical code. (Please run macro/micro-benchmarks and show the results.)

- [ ] My PR contains critical fixes that are necessary to be merged into the latest release. (Please check out the [details](https://github.com/risingwavelabs/risingwave/blob/main/CONTRIBUTING.md))

## Documentation

- [ ] My PR needs documentation updates. (Please use the **Release note** section below to summarize the impact on users)

## Release note

If this PR includes changes that directly affect users or other significant modifications relevant to the community, kindly draft a release note to provide a concise summary of these changes. Please prioritize highlighting the impact these changes will have on users.

</details>

### CODE REVIEW

File: grafana/risingwave-dev-dashboard.dashboard.py [hunk 1]
```diff
@@ -3919,6 +3921,60 @@ def section_network_connection(outer_panels):
...
 
+stream_graph_extra_json = {
+    "options": {
+        "useBackground": True,
+        "nodeSize": {"minWidth": 30, "minHeight": 30},
+        "legend": {
+            "show": False,
+            "stats": ["mean", "last", "min", "max", "sum"],
+        },
+        "contentUrl": "http://localhost:5691/api/flowchart/table/${table}",
```
Comment (fuyufjh) [inline grafana/risingwave-dev-dashboard.dashboard.py:3932]: Cool! In this way the Grafana can display a dynamic graph, generated by the Rust code

Comment (BugenZhao) [reply grafana/risingwave-dev-dashboard.dashboard.py:3932]: Yeah, but I find limitations. 😕 The requests seem to be sent from the browser, so the connectivity might be a problem.

Comment (BugenZhao) [reply grafana/risingwave-dev-dashboard.dashboard.py:3932]: The JSON data source might help to fetch the flowchart content, which supports sending requests from the server:

https://github.com/simPod/GrafanaJsonDatasource

**Comment by BugenZhao** [comment]
Superseded by #14446.

**Pull request closed without merging**